test-workloads/src/bin/guestvm.rs

```diff
@@ -176,6 +176,18 @@ pub fn test_vector() {
 const TEST_CSR: &[u8] = include_bytes!("test-ed25519.der");
 
 fn test_attestation() {
+    let caps = attestation::get_capabilities().expect("Failed to get attestation capabilities");
```

Comment:
Should we execute the SBI probe below first before panicking?

Reply:
Yep, good catch.